server/routes/shopId.js

```diff
@@ -0,0 +1,56 @@
+const Busboy = require("busboy");
+const express = require("express");
+const router = express.Router();
+const AWS = require("aws-sdk");
+const verify = require('./verifyToken');
+
+function uploadToS3(file) {
+  let s3bucket = new AWS.S3({
+    accessKeyId: process.env.IAM_USER_KEY,
+    secretAccessKey: process.env.IAM_USER_SECRET,
+    Bucket: process.env.BUCKET_NAME
+  });
+  let imageData;
+  s3bucket.createBucket(function() {
```

Comment:
don't we need to just run this once? We shouldn't have to run this everytime we upload to S3?